src/main/java/github/scarsz/discordsrv/DiscordSRV.java

```diff
@@ -1728,6 +1728,77 @@ public void sendLeaveMessage(Player player, String quitMessage) {
         }
     }
 
+    
+    /**
+     * Gives an "online" role to a player after joining. The role id can be added in the config.
+     *
+     * @param player the player
+     * @see #removePlayerOnlineRole(Player)
+     */
+    public void givePlayerOnlineRole(Player player) {
+        if (player == null) throw new IllegalArgumentException("player cannot be null");
+
+        // retrieves the role if possible, if not returns null
+        String roleName = DiscordSRV.config().dget("OnlineRoleName").asString();
+        Role role = DiscordUtil.getRoleByName(getMainGuild(), roleName);
+        if (role == null){
+            getLogger().severe("Role \""+roleName+"\" could not be found. Make sure OnlineRoleName is correctly set in the config.yml");
```

Comment:
Wrong logging method, add space around parts of the string

Reply:
for logging should I use error(), warning(), or something else?

Reply:
My gripe was to use the static methods DiscordSRV has such as `DiscordSRV#info`

Reply:
Yeah I eventually found those. Is `#info` the correct one to use? I feel like warning would make more sense. Your call, ofc

Reply:
Info was just an example. I'd probably use warning for this myself.